src/module.c

```diff
@@ -4077,6 +3978,7 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *     * `!` -- Sends the Redis command and its arguments to replicas and AOF.
  *     * `A` -- Suppress AOF propagation, send only to replicas (requires `!`).
  *     * `R` -- Suppress replicas propagation, send only to AOF (requires `!`).
+ *     * `3` -- Return the reply in resp3.
```

Comment:
```suggestion
 *     * `3` -- Return the reply in RESP3. This will change the command reply.
 *              e.g. HGETALL returns a map instead of flat array, 
```